ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3087,3 +3087,86 @@ def test_tensorflow_Pack(
         values=x,
         axis=axis,
     )
+
+
+@st.composite
+def _pad_helper(draw, *, padding_types=None, return_constant_values=False):
+    if padding_types is None:
+        padding_types = ["int32", "int64"]
+
+    dtype, input, shape = draw(
+        helpers.dtype_and_values(
+            min_num_dims=1,
+            ret_shape=True,
+        )
+    )
+    ndim = len(shape)
+    padding_dtype, paddings = draw(
+        helpers.dtype_and_values(
+            available_dtypes=padding_types,
+            shape=(ndim, 2),
+            min_value=0,
+            max_value=10,
+        )
+    )
+
+    if return_constant_values:
+        _, constant_values = draw(
+            helpers.dtype_and_values(
```

Comment:
we simply can use the `int` strategy here, this is causing the test to fail in `PyTorch` as we're generating a `0-dim` array of type `bfloat16`, which isn't the expected input for this parameter.

Reply:
I don't think that `int` strategy is suitable here, as we need a constant value with the same type of the original tensor, hence why I used `dtype_and_values` instead of a simple strategy while I need a simple scalar.

Reply:
I see! completely missed this one, my bad! I'll have another look to see why it's exactly failing